### DISCUSSION TITLE
Ways to donate & possibility to integrate animations in commercial trainings ?

### DISCUSSION THREAD

**Discussion by fcamblor** [pcottle/learnGitBranching#1111] [Ideas]
Hello Peter,

First of all, you've made a wonderful website which is very useful to use when training people at using Git.

I even discovered that we could contribute levels by ourselves, that looks very cool !

I was wondering... I didn't see any Github Sponsorship button on your profile: could you consider adding it so that I can praise my company to share some bounty with you for all the hard work ?

Also, I was wondering... would it be ok (from a license point of view, but also from your point of view) to put some videos about learngitbranching in our (commercial) training slides, because the animations around cherry-pick, rebase, merge and so on are so well done that it would be far better than doing it ourselves with google slides animations :-)
Both you and the website would obviously be credited in our training material.

Thanks again for this great tool !

**Comment by fcamblor** [comment]
Sorry, seems like first part of the questions seems related to #969 (but I reiterate it anyway, because you really deserve it :-) )

**Comment by pcottle** [comment]
> Also, I was wondering... would it be ok (from a license point of view, but also from your point of view) to put some videos about learngitbranching in our (commercial) training slides, because the animations around cherry-pick, rebase, merge and so on are so well done that it would be far better than doing it ourselves with google slides animations :-)

Totally ok to do this! 😇 It'd be nice if you could add attribution back to the project (just showing Learn Git Branching somewhere on the slide) but if it's too difficult to do that, it's ok to omit it.

Alright I went ahead and kicked off my application process for sponsorship. It can take a few days to get approval, but we can use any funds collected to improve the website!
<img width="826" alt="Screenshot 2023-11-09 at 10 17 42 AM" src="https://github.com/pcottle/learnGitBranching/assets/1135007/64a7e6e4-6425-47f2-a276-92e10208f4d3">

**Reply by fcamblor** [reply]
Great ! 🚀 

> Totally ok to do this! 😇 It'd be nice if you could add attribution back to the project (just showing Learn Git Branching somewhere on the slide)

Sure !

**Comment by pcottle** [comment]
Sponsorship is approved! Should be a button at the top of the project you can click 😇
